source/features/fix-first-tab-length.css

```
@@ -0,0 +1,18 @@
+/* Remove diff signs from the layout flow */
+.file .blob-code-inner::before,
+.file .blob-code-marker-cell::before {
+	position: absolute;
+	transform: translateX(-100%);
+}
+
+/* Restore the space that collapsed as a consequence of the previous rule */
+.file :not(.js-suggested-changes-blob) tr:not(.js-expandable-line) .blob-code,
+.file .js-suggested-changes-blob .blob-code-inner {
```

The first selector doesn't exclude `.js-suggested-changes-blob` as expected because there are many elements between `.file` and `tr`. This means it can be dropped.


```suggestion
.file tr:not(.js-expandable-line) .blob-code, /* Each line of code except `@@ ... @@` headers */
.file .js-suggested-changes-blob .blob-code-inner { /* Each line of code in PR review comments */
```